docs/source/tutorials/odes/odesolvers_from_scratch.ipynb

```diff
@@ -56,8 +56,8 @@
    "metadata": {},
```

Comment:
<p>This figure (or at least the preview I see here) seems incomplete. Though, you specifically select only the first dimension above, so that might be intentional.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>